include/neml2/tensors/Scalar.h

```diff
@@ -47,6 +47,13 @@ class Scalar : public FixedDimTensor<1, 1>
   /// Negation
   Scalar operator-() const;
 
+  // @{ In-place operations
```

Comment:
Don't you need the three slashes for this to be picked up correctly?

Reply:
whoops yes